### ISSUE TITLE
Unable to move after loading level

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/portal2#67]
In Chapter 6, in the level after opening the hatch, I was unable to move. I then saved and reloaded the game and then it worked.

**Comment by jacobmischka** [comment]
Are you referring to the part where you plug Wheatley into the wall and he opens a hatch in the wall? I got stuck there once my very first time, and have been unable to reproduce since, even though I tried several times.

**Comment by ghost** [comment]
No, I was referring to the part after the large room with 2 cabins on either side, there you need to activate a timed switch in each cabin using portals and if you do that it will open a (very large) hatch that lets you to the next level.

**Comment by jeremyblanchard** [comment]
Same happened to me on PS3
